.github/workflows/full_render_doc-en_test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    name: doc-en full render test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PhD
+        uses: actions/checkout@v2
```

Comment:
Normal this uses the V2 version?

Reply:
You're right, it should probably be v4. I copied it from an other test. :-)